app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -7,6 +7,100 @@ def available_vms
     availability_zone.vms
   end
 
+  def self.params_for_create(ems_id)
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'size',
+          :id         => 'size',
+          :label      => __('Size (in bytes)'),
+          :type       => 'number',
+          :step       => 1024 * 1024 * 1024,
+          :isRequired => true,
+          :validate   => [{:type => 'required'}],
+        },
+        {
+          :component => 'select',
+          :name      => 'volume_type',
+          :id        => 'volume_type',
+          :label     => _('Cloud Volume Type'),
+          :condition => {
+            :when => 'edit',
+            :is   => false,
+          },
+          :options   => ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolumeType.where(:ext_management_system_id => ems_id).map do |cvt|
+            {
+              :label => cvt.name,
+              :value => cvt.value,
```

Comment:
I'm trying to understand these - does `cvt.value` get set from the `cvt.name` value?

As I look more closely at cloud_volume_types table I'm thinking it would be best to set:
```
:label => cvt.description,
:value => cvt.name,
```

Reply:
Yeah, the `value` is incorrect, it should be `type` instead...at least compared with OpenStack.